src/colours.js

```diff
@@ -155,6 +155,22 @@ const Colours = {
       }
     }
   },
+  registerBlockStyles: function () {
```

Comment:
As I understand it, this finds all of the block styles in `cssColours` (where a block style has a `primary` property) and creates a corresponding stanza in the current Blockly theme.

I think it would be better to [create a new theme](https://developers.google.com/blockly/guides/configure/web/appearance/themes) based on the `zelos` theme and add these colours to it before injection. That way you aren't relying on modifying an object that's already in use, and you can pass the correct theme into the call to `inject`.

Reply:
Done!